### DISCUSSION TITLE
2023/05/08 Task

### DISCUSSION THREAD

**Discussion by hyunnnn98** [2P3S/scrum-dice#51] [General]
## 승현

- [x] #50
- [x] #9 
   - 멤버아이디 로컬 스토리지 저장 
- [ ] #52 

## 재순

- [x] https://github.com/2P3S/2p3s-backend-system/issues/9
	- 채팅방 입장 성공 / 실패
	- 입장 후 메시지 보내는 것
- [ ] API, Socket 모듈화
- [ ] Document 작성

## 승형

- [ ] https://github.com/2P3S/2p3s-backend-system/issues/7
    - 사용자 접속여부를 Local Storage로 구분하는 로직 설계
- [ ] Api, Socket 진행 중인 코드 분석하면서 Scrum dice에 어떻게 적용할 것 인지 (필요한 데이터 정리)

**Comment by JeongJaeSoon** [comment]
**멤버아이디 로컬 스토리지 저장**
* 재검토 필요
* 굳이 로컬 스토리지 저장할 필요가 있는가?
* 오랜 시간이 지난뒤에 재접속했을때에도 다시 접속하고 싶을수도?
* 방마다 관리하는지? 아니면 사용자 PC 에 하나만 하는지?
* 계속 쌓아둘 경우, 불필요한 데이터가 쌓인다

**Comment by JeongJaeSoon** [comment]
**카드 UI**
* 옵션 부분 이름 지우고 오른쪽에 설명 달기
* 카드 사이즈는 4열로 두는게 좋을듯..? 가능하면 폭도 맞추고 싶다
* 모바일 UI 도 슬슬 FIX 해야함

**Comment by hyunnnn98** [comment]
**채팅방 Socket 기초 작성**

- namespace 관련 소스코드 검토 ( by. @LeeJuYong0909  )
- postman 작동법 설명
